tests/integration/test_snapshot_option_defaults.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+
+@pytest.fixture
+def testcases():
+    return {
+        "inject": (
+            """
+            def test_injection(snapshot):
+                assert snapshot is not None
+            """
+        ),
+    }
+
+
+@pytest.fixture
+def run_testcases(testdir, testcases):
+    pyfile_content = "\n\n".join(testcases.values())
+    testdir.makepyfile(test_file=pyfile_content)
+    return testdir.runpytest("-v")
+
+
+def test_injected_fixture(run_testcases):
```

Comment:
I don't understand. What are we testing here? Just that the fixture works? Isn't that technically covered by other tests? Or is this just a more explicit test?